### PR TITLE
JS3/W3  PR Added

### PR DESCRIPTION
**Pull request by Elia-Git**
JS3/W3 Homework

### CODE REVIEW

File: javascript/javascript3/week3/Canvas/draw-circles.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+console.log("loadeddd");
+//USING JS
+let canvas = document.getElementById("myCanvas");
+
+let context = canvas.getContext("2d");
+
+function drawSeveralCircles() {
+    // Draw several circles
+    for (var i = 0; i < 10; i++) {
```
Comment (sowmya1408) [inline javascript/javascript3/week3/Canvas/draw-circles.js:9]: Var is not good practice to use. As per ES6 specifications use either const or let, depending on the variable used.

File: javascript/javascript3/week3/Canvas/draw-circles.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+    draw() {
+        context.beginPath();
+        context.arc(this.x, this.y, this.r, this.startAngle, this.endAngle);
+        context.fillStyle = this.fillColor;
+        context.fill();
+        context.strokeStyle = "yellow";
+        context.stroke();
+    }
+}
```
Comment (sowmya1408) [inline javascript/javascript3/week3/Canvas/draw-circles.js:67]: perfect 💯

File: javascript/javascript3/week3/PromiseAll/repositories.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+const fetchAll = (userName) =>
+    fetch(userUrl + userName)
+    .then((response) => response.json())
+    .then((data) => data.items);
+
+const usersPromises = [
+    fetchAll("anilMula"),
+    fetchAll("merihnguse"),
+    fetchAll("Deepti-Dk"),
+    fetchAll("benna100"),
```
Comment (sowmya1408) [inline javascript/javascript3/week3/PromiseAll/repositories.js:18]: Good to see that you have stored all functions in to a variable and then use it at Promise.all 👏

**Review by sowmya1408 — APPROVED** [review]
Awesome work 👏 
Just a small feedback , it good to know the difference between var, let and const: There are places were you could still use const but have used let.

**Comment by Elia-Git** [comment]
All noted, thank you!

**Pull request merged**